### PR TITLE
Updated to Magento 1.9.4.2

### PR DESCRIPTION
**Pull request by JeroenBoersma**
+ downloaded latest release and merged into mainstream

Howto compare

* download archive
* rm all files
* extract into repository
* commit and compare

### CODE REVIEW

File: lib/phpseclib/PHP/Compat/Function/bcpowmod.php
Comment (Flyingmana) [inline lib/phpseclib/PHP/Compat/Function/bcpowmod.php:17]: note: interesting to look into later

**Comment by Flyingmana** [comment]
did not compare with a download yet, but looks already good

**Comment by rvelhote** [comment]
Is this repository installable with composer? We were using https://github.com/firegento/magento but it now points to this one (we are not ready to use LTS version yet).

**Comment by EricSeastrand** [comment]
I have verified using diff: this version matches the download from the official Magento site.

If your update scripts, like mine, depend on having this code in a version-specific repo tag, feel free to use my repository. I created a new fork, merged in this PR from @JeroenBoersma , and released 1.9.4.2 tag.

https://github.com/willcodeforfood/magento-mirror/tree/1.9.4.2

**Comment by davidwindell** [comment]
Sorry to chase but can someone please release this? We have customers chasing us :)

**Comment by Flyingmana** [comment]
@davidwindell we still need at least one other approving Review, if one from your company could spend an hour to verify, this is the official one, this would help a lot

**Comment by cundd** [comment]
Can everybody do these reviews or do one has to be part of the OpenMage organization?

**Comment by davidwindell** [comment]
@Flyingmana We've just tested this here at @outeredge by following the steps above, can confirm no difference in the archive to the pull request.

`nothing to commit, working tree clean`

**Comment by rvelhote** [comment]
@Flyingmana I can also confirm that the PR matches all files from the official release.

1. Cloned `JeroenBoersma/magento-mirror`
2. Downloaded the official release from magento.com
3. Deleted all files from the cloned repository
4. Copied the contents from the official package into the (now empty) repository
5. Confirmed that there are no modified files between the PR and the official release

Also it would be super if someone from OpenMage could add a `composer.json` file to this repository and submit it to packagist (much like the old firegento repository).

**Comment by NikoGrano** [comment]
@rvelhote Mirror is a mirror. Not a option. Sorry.

**Comment by davidwindell** [comment]
@Flyingmana do you have everything need to merge now?

**Comment by EricSeastrand** [comment]
@rvelhote While it isn't an option to add a `composer.json` to *this* mirror, do note that you can simply fork this repo and add a `composer.json` to your fork. When a new version is released to the default branch of this repo, you can simply merge that update from this upstream repo into your own fork with a command like `git pull https://github.com/OpenMage/magento-mirror.git`. You won't even have to fix any merge conflicts if your only change was to add `composer.json` to your repo.
This is actually the strategy we've used for years to update our Magento stores while keeping customization intact: put it all in a git repo, and let git merge in the updates for us. I only once had to merge something manually, and it was because the core file was modified by a patch. So if all you do is add composer.json, it should always work flawlessly.

**Review by Flyingmana — APPROVED** [review]

**Comment by Flyingmana** [comment]
Thank you all for helping 👍

**Comment by BassemN** [comment]
@Flyingmana Please tag `1.9.4.2`

**Comment by Flyingmana** [comment]
done ✔️

**Pull request merged**
